### ISSUE TITLE
Fail to go get

### ISSUE THREAD

**Issue by yukihiko-shinoda** [hashicorp/packer#10291]
#### Overview of the Issue

Until yesterday (2020-11-20), I could `go get github.com/hashicorp/packer`.
Today (2020-11-21), I can't `go get github.com/hashicorp/packer`.
It looks required to develop packer plugin:

cf. [Plugins - Extending | Packer by HashiCorp](https://www.packer.io/docs/extending/plugins#plugin-development-basics)

Or, do I mistake something to prepare develop environment for packer plugin?
(I'm quite new to develop it...)

#### Reproduction Steps

1\.
Create Dockerfile:

```docker
FROM golang
RUN go get github.com/hashicorp/packer
```

2\.
Run build:

```console
docker build .
```

### Packer version

`master` branch

### Simplified Packer Buildfile

Not related with this issue.

### Operating system and Environment details

Windows 10 + Docker Desktop + WSL2 Ubuntu 20.04

### Log Fragments and crash.log files

```console
Step 2/6 : RUN go get github.com/hashicorp/packer
 ---> Running in f35a2cb02067
# github.com/hashicorp/packer/provisioner/salt-masterless
src/github.com/hashicorp/packer/provisioner/salt-masterless/provisioner.go:401:29: undefined: provisioner
ERROR: Service 'golang' failed to build : The command '/bin/sh -c go get github.com/hashicorp/packer' returned a non-zero code: 2
```

The following line seems to had been merged yesterday:

```golang
	if p.config.GuestOSType == provisioner.WindowsOSType {
```

cf: #10201

**Comment by azr** [comment]
Hello there ! Thanks for opening, I fixed this in #10296.

**Comment by yukihiko-shinoda** [comment]
Thanks for your quick fix and teaching how to avoid, I got it🙂
